source/includes/security/authentication.kt

```diff
@@ -0,0 +1,206 @@
+import com.mongodb.*
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.BsonInt64
+import org.bson.Document
+
+// SCRAM Authentication
+// start-default-cred-string
+val mongoClient =
+    MongoClient.create("mongodb+srv://<db_username>:<db_password>@<hostname>:<port>/?authSource=<authenticationDb>")
```

Comment:
Hi @vbabanin, I opted for the SRV URI format for the connection string examples because I get the following timeout error using the standard format. It is because ssl is not enabled:

`com.mongodb.MongoTimeoutException: Timed out while waiting for a server that matches ReadPreferenceServerSelector{readPreference=primary}. Client view of cluster state is {type=UNKNOWN, servers=[{address=atlascluster.spm1ztf.mongodb.net:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketException: atlascluster.spm1ztf.mongodb.net}, caused by {java.net.UnknownHostException: atlascluster.spm1ztf.mongodb.net}}]`

For the same reason, I used `builder.srvHost("<hostname>")` instead of the alternative. In order to make to the examples copyable and runnable, does it make sense to use the +srv format? Thanks!